### ISSUE TITLE
OSM roads to align to these

### ISSUE THREAD

**Issue by justinelliotmeyers** [facebookmicrosites/Open-Mapping-At-Facebook#23]
**Title:** Title of the issue here
OSM roads to align to these

#
**Files:** 

List the files related to this issue
https://mapwith.ai/country_exports/CO_mapwithai_road_data.gpkg.tar.gz

#
**Description:**

Description of the problem or suggestion on the documents. Feel free to add screenshots.
can you start to upload the osm roads that are the differnce of these? So when we go to merge the data for a complete picture, we don't get duplicates?

Basically asking for one country with 100% coverage - x flagged as osm and y flagged as meta.

If this doesn't make sense let me know, but I am trying to get the complete picture painted and not try to mash these together weeks later and have tons of duplicates
